src/thing_creature.c

```diff
@@ -1241,19 +1255,162 @@ void process_thing_spell_teleport_effects(struct Thing *thing, struct CastedSpel
         if (thing_in_wall_at(thing, &pos))
         {
             const struct Coord3d* newpos = NULL;
+            struct Coord3d room_pos;
             {
```

Comment:
I know you didn't place it, but is this bracket doing anything here? It looks like a mistake to me.